Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN ln -sfT container_settings.py /app/squad/local_settings.py && \
     python3 -m squad.frontend && \
     ./manage.py collectstatic --noinput --verbosity 0 && \
     ./manage.py compilemessages && \
-    python3 setup.py develop && \
+    python3 setup.py develop --no-deps && \
```

Comment:
How did you figure out this option? I didn't find this at https://setuptools.readthedocs.io/en/latest/setuptools.html#develop-deploy-the-project-source-in-development-mode

Reply:
`python3 setup.py develop --help`